### ISSUE TITLE
Case/Seg - Tally table - show term names for HPO IDs

### ISSUE THREAD

**Issue by wrightmw** [ClinGen/clincoded#1917]
When HPO IDs are added then the term name should also be shown in the Tally table:

![Screen Shot 2019-05-23 at 6 49 03 PM](https://user-images.githubusercontent.com/15131169/58297138-78ac6180-7d8b-11e9-9ed2-b5d4d800e1e3.png)

e.g. e.g. HP:0010704 would be 1-2 finger syndactyly (HP:0010704)

**Comment by markmandell** [comment]
**Steps to test:**

1. New Variant Curation, start an Interpretation
2. Navigate to Case/Segregation panel, save evaluation for any criteria, and click "Add Evidence" to bring up modal
3. Enter any PMID, preview and click "Next"
4. Enter a comma separated list of HPO IDs for the "Phenotypic feature(s) associated with proband(s) (HPO) (PP4)" input (i.e. HP:0010010, HP:0010704, HP:0030300), use Get Terms button
  * Form validation present for HPOs via "Get Terms" and "Submit", cannot fetch or save unless HPO ID format is correct and is a comma separated list. 
  * Terms list and IDs in input have to match, if not, form error is triggered.
  * Get Terms button fetches the terms for the respective IDs and saves as an object to be passed to Case/Segregation tables, terms are cleared using red "Clear Terms" button
  * Duplicates are omitted

5. Feel free to fill in another inputs, otherwise, click "Submit". Redirected to Case/Segregation panel, IDs with the respective terms are displayed in gray "Master Table" cell and in "Curated Evidence (Specificity of phenotype)" towards bottom of page.

6. Edit the Evidence in either table, and the terms should still be displayed to the right of input, "Get Terms" button still disabled.

For testing of upgrade script in release candidate, here are some variant names with existing interpretations w/ Case/Seg PP4 criteria:

NM_000546.5(TP53):c.554G>A (p.Ser185Asn)
NM_000018.4(ACADVL):c.1316G>A (p.Gly439Asp)
NM_005105.4(RBM8A):c.-21G>A

**Comment by cgpreston** [comment]
@markmandell, thanks! I tested this on the R33 candidate,  
1. I see the HPO IDs and names when I add them to the case seg tab. 
2. I tested a few scenarios: including adding the HPO terms to the new non-published evidence categories and in. the context of adding to the evidence pool for an existing variant. I also tested adding 1 and multiple HP terms. 
3. I saw the information appear as expected in the tally tables, the hoverover and the, specificity of phenotype tables.
